### PR TITLE
Movie Project - RAGNA

### PR DESCRIPTION
**Pull request by Ragtulf**
Link to my deployed project:
[https://condescending-goodall-d94da0.netlify.com/](https://condescending-goodall-d94da0.netlify.com/)

My reflections on how this week's project turned out:
It went really well and was a fun project.

Things I'd like to have clarified or explained in more detail:
How to get and show e.g. the backdrop image in a size fitting for the device where you are looking at the website.

### CODE REVIEW

File: code/src/pages/MovieDetails.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        console.log(json)
+        setMovie(json)
+      })
+  }, [movieId])
+
+  return (
+    <div className='backdrop' style={{ backgroundImage: `url(https://image.tmdb.org/t/p/original/${movie.backdrop_path})` }}>
+      <div className='backdrop-gradient'>
+        <div className='content-container'>
+          <img className='details-poster' src={`https://image.tmdb.org/t/p/w500/${movie.poster_path}`} alt={movie.title} />
```
Comment (ericaglimsholt) [inline code/src/pages/MovieDetails.js:25]: Nice use of the alt!

File: code/src/pages/MovieDetails.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            <p className='overview'>{movie.overview}</p>
+          </div>
+        </div>
+      </div>
+      <Link to='/'>
+        <div className='back-section'>
+          <BackButton />
+          <h4 className='hvr-forward'>Back</h4>
+        </div>
+      </Link>
```
Comment (ericaglimsholt) [inline code/src/pages/MovieDetails.js:38]: Hmm... I would think that the `BackButton` component would have the `Link` to it aswell?

File: code/src/pages/MovieDetails.js (hunk 1 above)
Comment (Ragtulf) [reply code/src/pages/MovieDetails.js:25]: Thank you! Although, I can't take credit. The idea came from a codealong from Damien :)

File: code/src/pages/MovieDetails.js (hunk 2 above)
Comment (Ragtulf) [reply code/src/pages/MovieDetails.js:38]: The `BackButton` component and the `h4` are wrapped in a `div` that is wrapped in a `Link`

**Pull request closed without merging**